modules/analysis-common/src/test/java/org/elasticsearch/analysis/common/CommonAnalysisPluginTests.java

```diff
@@ -116,4 +119,44 @@ public void testEdgeNGramNoDeprecationWarningPre6_4() throws IOException {
             assertNotNull(tokenFilterFactory.create(tokenizer));
         }
     }
+
+    /**
+     * Check that the deprecated analyzer name "standard_html_strip" issues a deprecation warning for indices created since 6.5.0
+     */
+    public void testStandardHtmlStripAnalyzerDeprecationWrning() throws IOException {
+        Settings settings = Settings.builder().put(Environment.PATH_HOME_SETTING.getKey(), createTempDir())
+            .put(IndexMetaData.SETTING_VERSION_CREATED, VersionUtils.randomVersionBetween(random(), Version.V_6_5_0, Version.CURRENT))
+            .put("index.analysis.analyzer.custom_analyzer.type", "standard_html_strip")
+            .putList("index.analysis.analyzer.custom_analyzer.stopwords", "a", "b")
+            .build();
+
+        IndexSettings idxSettings = IndexSettingsModule.newIndexSettings("index", settings);
+        try (CommonAnalysisPlugin commonAnalysisPlugin = new CommonAnalysisPlugin()) {
+            IndexAnalyzers analyzers = createTestAnalysis(idxSettings, settings, commonAnalysisPlugin).indexAnalyzers;
+            Analyzer analyzer = analyzers.get("custom_analyzer");
+
+            assertNotNull(((NamedAnalyzer) analyzer).analyzer());
+            assertWarnings(
+                "Deprecated analyzer [standard_html_strip] used, replaced by using [html_strip] char_filter");
+        }
+    }
+
+    /**
+     * Check that the deprecated analyzer name "standard_html_strip" issues a deprecation warning for indices created before 6.4.0
+     */
+    public void testStandardHtmlStripAnalyzerNoDeprecationPre6_5() throws IOException {
```

Comment:
This test doesn't seem to do what the comment suggests it should do?  I think it can be repurposed to check that an exception is thrown if you try to create an analyzer of type `standard_html_strip` with an index created version greater than or equal to 7.0, and the test above can just check that the warning is emitted on any version before 7.0

Reply:
Ah, good catch. I forgot add "does NOT" after copy & paste...

I'm not sure your comment about 7.0. Is your comment about PR for master branch?

Reply:
Ah, I hadn't noticed that this was against 6.x.  Do you have a separate PR against master?  I think it makes most sense to do this against master and then backport?

Reply:
Ah, I made this PR for logging deprecation and I thought we removeed this analyzer in 7 at that time. 

I agree to do against master first. I will change this PR to against 7. sorry